service/spawn.go

```diff
@@ -176,6 +176,14 @@ func (uis *UIServer) modifySpawnHost(w http.ResponseWriter, r *http.Request) {
 			uis.WriteJSON(w, http.StatusBadRequest, fmt.Sprintf("Host %v is already terminated", host.Id))
 			return
 		}
+		if host.Status == evergreen.HostUninitialized {
+			if err := host.SetTerminated(); err != nil {
```

Comment:
Will this actually terminate the host though?

Reply:
You can only hit the error if we haven't tried to start the host on EC2 yet, so there isn't anything to terminate. This will clean it up in the DB